code/modules/client/preferences.dm

```diff
@@ -192,13 +197,10 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	return data
 
 /datum/preferences/ui_assets(mob/user)
-	var/list/assets = list(
-		get_asset_datum(/datum/asset/spritesheet/preferences),
-		get_asset_datum(/datum/asset/json/preferences),
-	)
+	var/list/assets = ..()
```

Comment:
There is no `..()` for `/datum/ui_assets`--we shouldn't do this because it makes it less obvious what is actually going on here